website/content/sdk/guides/intros/stocks/options.md

```diff
@@ -1,82 +1,110 @@
-excerpt: "This guide introduces the Stock Options module, within the context of the OpenBB SDK"
-geekdocCollapseSection: true
 ---
-The Stock Options module of the OpenBB SDK wraps the functions of the Options sub-menu within the [OpenBB Terminal](/terminal/guides/intros/stocks/options). This allows the user greater flexibility when researching and anlayzing options data. Start a Python script of Notebook file by importing the dependencies:
 
-```python
-from openbb_terminal.sdk import openbb
-
-import pandas as pd
-import numpy as np
-
-import matplotlib.pyplot as plt
-
-%matplotlib widget
-```
+Please be advised that this module is subject to improvements, the guide here may not reflect its current status.
 
 ## How to Use
 
-All functions within the Options module are printed with: `openbb.stocks.options`
-
-![Options Module Functions](https://user-images.githubusercontent.com/85772166/199891549-1331a65c-6251-4206-b77c-c64dfabe70f4.png "Options Module Functions")
-
-The [Unusual Options function](/sdk/reference/stocks/options/unu/) returns a Tuple containing the DataFrame and a timestamp from when the list was last updated.
+The functions in the Options sub-module are listed below, along with a short description.
+
+|Path |Description |
+|:----|-----------:|
+|openbb.stocks.options.chains |Options Chains |
+|openbb.stocks.options.dte |Convert a Date in the Future to DTE |
+|openbb.stocks.options.expirations |List of Expiration Dates for the Underlying |
+|openbb.stocks.options.grhist |Historical Greeks for Individual Options |
+|openbb.stocks.options.hist |Historical Data from Tradier |
+|openbb.stocks.options.hist_ce |Historical Data from ChartExchange |
+|openbb.stocks.options.info |Options Statistics for the Underling from BarChart |
+|openbb.stocks.options.last_price |Last Price of the Underlying from Tradier |
+|openbb.stocks.options.pcr |Historical Put/Call Ratio |
+|openbb.stocks.options.screen |Options Screener |
+|openbb.stocks.options.unu |Unusual Options |
+|openbb.stocks.options.vsurf |Volatility Surface |
+
+Alternatively, the contents of the menu or a function's docstrings can be printed using Python's built-in help.
 
 ```python
-unusual_options,timestamp = openbb.stocks.options.unu()
-unusual_options
+help(openbb.stocks.options)
 ```
 
-![Unusual Options](https://user-images.githubusercontent.com/85772166/199891691-da3af87e-99c3-4d01-881f-62b6b0d3ae34.png "Unusual Options")
+## Examples
 
-Get up to ten years of historical put/call ratios (US equity options only), with the `PCR` function:
+To reduce the amount of characters in a script, try importing the module as shown below.
+
+### Import Statements
 
 ```python
-pcr_spy = openbb.stocks.options.pcr(symbol = 'SPY', start_date = '2012-11-03')
-pcr_spy.rename(columns = {'PCR': 'Put/Call Ratio'})
+from openbb_terminal.sdk import openbb
+import pandas as pd
+opt = openbb.stocks.options
```

Comment:
Nowhere in any guide we mention creating a short code for the module so I'd advise not adding it here. It also makes the rest of the examples confusing if you read over this.

Reply:
done